src/__tests__/ReduxRouter-test.js

```diff
@@ -223,8 +223,6 @@ describe('<ReduxRouter>', () => {
       'or a class/function (for composite components) but got: object. ' +
       'Check the render method of `ReduxRouterContext`.';
 
-    const routingContextInvalidPropErrorMessage = 'Invalid prop `RoutingContext` of type `object` supplied to `ReduxRouterContext`';
```

Comment:
I wanted your opinion on the matter, because you added the test. I don't want to blindly remove stuff from tests, just to make them pass.

Sorry to waste your precious time 😒